oaiharvest/metadata.py

```diff
@@ -45,9 +45,10 @@ def writeMetadata(self, metadata_prefix, element, metadata):
 class XMLMetadataReader(object):
     """Really simple MetadataReader to serialize metadata to pretty XML."""
     def __call__(self, metadata_element):
-        return '\n'.join([tostring(rec_element,
-                                   method="xml",
-                                   pretty_print=True
-                                   )
-                          for rec_element
-                          in metadata_element])
+        # Six call fixes 'sequence item 0: expected str instance, bytes found'
+        return ('\n' if six.PY2 else b'\n').join(
```

Comment:
If `lxml.etree.tostring()` won't return unicode (i.e. the standard string type) in Python3, then would it be nicer to have:

```python
    return six.binary_type('\n').join(...)
```

This makes it very explicit that we're dealing with byte strings.